### PR TITLE
HLD - OpenConfig support for ARP/NDP dump using management framework interfaces (CLI/REST API/gNMI)

### PR DESCRIPTION
**Pull request by venkatmahalingam**
Draft HLD - Openconfig support for ARP/NDP dump using management framework interfaces (CLI/REST API/gNMI)

### CODE REVIEW

File: doc/mgmt/SONiC_OC_ARP_NDP_get_support_HLD.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+| show ipv6 neighbors summary | IS-CLI drop-in replace |  | |
+| show ipv6 neighbors interface { ethernet/port-channel/vlan } |IS-CLI drop-in replace | | |
+|show ipv6 neighbors interface { ethernet/port-channel/vlan } summary  | IS-CLI drop-in replace | | |
+|show ipv6 neighbors ``<A::B>``  | IS-CLI drop-in replace | | |
+|show ip arp mac-address ``<mac>`` | SONIC | | In order to match ARP command options, having mac-address based filter for this command as well|
+
+
+
+
+### 3.6.3 REST API Support
```
Comment (kkvedula) [inline doc/mgmt/SONiC_OC_ARP_NDP_get_support_HLD.md]: Will SONIC YANG be supported as well?

**Comment by kkvedula** [comment]
Will SONIC YANG be supported as well?

**Comment by jeffyin-dell** [comment]
@venkatmahalingam -- what do you want to do with this PR? close it?

**Comment by obaidamin** [comment]
@jeff-yin I went through the HLD and it seems fine. Can you please merge this pull request after reviewing it?

**Review by bhavini-gada — APPROVED** [review]

**Pull request closed without merging**